### ISSUE TITLE
BAT files no longer dumps to VS Output window "vsSolutionBuildEvent"

### ISSUE THREAD

**Issue by mef526** [3F/vsSolutionBuildEvent#34]
The previous version dumped the output of a batch file into the output window "vsSolutionBuildEvent". Now the bat file runs in a CMD window and the output is lost.
Attached is the .vssbe file with the command used
[Copy.vssbe.txt](https://github.com/3F/vsSolutionBuildEvent/files/490733/Copy.vssbe.txt)

**Comment by 3F** [comment]
Thanks for report,

The File mode is very old mode, and probably it broken after this changes: cde59ac904 :(

I will check it and to try to combine later

**Temporarily**, you can use, for example:

```java
string #[File sout(string filename [, string args [, integer timeout]])]
```

```java
#[OWP item("MyOutput").writeLine(true): mixed data]
```

e.g.:

```java

#[try
{ 
    #[OWP item("MyOutput").writeLine(true): #[File sout("fname.exe", "args"])]]
}
catch(err, msg)
{

}]
```
etc.

http://vssbe.r-eg.net/doc/Scripts/SBE-Scripts/Components/

**Comment by 3F** [comment]
Please check fixes from 85ad3b7 (debug versions [here](https://ci.appveyor.com/project/3Fs/vssolutionbuildevent/history))
